test/ATM/ATMFactorySetATMSettingsTest.js

```diff
@@ -0,0 +1,79 @@
+// JS Libraries
```

Comment:
Please, verify the folder is ```test/atm``` (lowercase) for all unit tests.

Reply:
Changed to lowercase